ReactNativeClient/lib/components/screens/note.js

```diff
@@ -523,7 +536,7 @@ class NoteScreenComponent extends BaseScreenComponent {
 
 		try {
 			if (mimeType == 'image/jpeg' || mimeType == 'image/jpg' || mimeType == 'image/png') {
-				const done = await this.resizeImage(localFilePath, targetPath, pickerResponse.mime);
+				const done = await this.resizeImage(localFilePath, targetPath, mimeType);
```

Comment:
Why was this changed?

Reply:
ImagePicker's response object does not have the mime field (see https://github.com/react-native-community/react-native-image-picker/blob/master/docs/Reference.md#the-response-object, I also verified in testing).
It's not a major issue, the only thing it affects is the extension of resized images.

Reply:
Good catch, that was indeed a bug!